qa/L0_java_memory_growth/test.sh

```diff
@@ -38,7 +38,8 @@ set -e
 
 MODEL_REPO=`pwd`/models
 SAMPLES_REPO=`pwd`/javacpp-presets/tritonserver/samples
-cp Simple.java $SAMPLES_REPO
+cp MemoryGrowthTest.java $SAMPLES_REPO/Simple.java
+sed -i 's/MemoryGrowthTest/Simple/g' $SAMPLES_REPO/Simple.java
```

Comment:
Why need to place it to simple repo as simple.java at the end? Can't it be compiled as is?

Reply:
Maven relies on the pom.xml in that folder, and everything's already set up that way to make it easy.

I originally did this the same way as the sequence test. To help with clarity around this comment, I modified the test script to instead copy it as MemoryGrowthTest.java and update the pom.xml instead (more lines of code, but should be clearer). I'm trying to keep everything standardized between tests.